FlowCrypt/Controllers/Compose/ComposeViewController.swift

```diff
@@ -634,6 +642,17 @@ extension ComposeViewController {
         }
     }
 
+    private func passwordNode() -> ASCellNode {
```

Comment:
let's use the term `messagePassword` so that we always know which one this is. So that would be `messagePasswordNode`